django/core/paginator.py

```diff
@@ -28,13 +28,30 @@ class Paginator:
     # Translators: String used to replace omitted page numbers in elided page
     # range generated by paginators, e.g. [1, 2, '…', 5, 6, 7, '…', 9, 10].
     ELLIPSIS = _("…")
-
-    def __init__(self, object_list, per_page, orphans=0, allow_empty_first_page=True):
+    default_error_messages = {
+        "page_not_an_integer": _("That page number is not an integer"),
+        "page_not_a_positive_integer": _("That page number is less than 1"),
+        "no_results": _("That page contains no results"),
+    }
```

Comment:
I'd use shorter keys, e.g.
```suggestion
    default_error_messages = {
        "invalid_page": _("That page number is not an integer"),
        "min_page": _("That page number is less than 1"),
        "no_results": _("That page contains no results"),
    }
```

Reply:
Great! Change applied!